styles/styles.css

```
@@ -649,6 +677,17 @@ body {
   border-color: #000000;
 }
 
+.search-finish {
+  font-family: 'PT Sans';
```

```suggestion
  font-family: 'PT Sans', sans-serif;
```
Критерий [CSS-06](https://up.htmlacademy.ru/htmlcss/35/criteries#CSS-06)